src/Command/ListMissingDownloadsCommand.php

```diff
@@ -122,8 +129,8 @@ private function getFixedUrl(string $url, string $release, int $format)
             }
 
             return 'redirect failed for ' . $url;
-        } catch (\Exception $e) {
-            return $e->getMessage();
+        } catch (\Exception $exception) {
+            return $exception->getMessage();
```

Comment:
This is an interesting question.  99% of the time I agree that single-character variables are unwise.  However, catching an exception named `$e` is so universally common that it seems wasteful and overly pedantic to spell it out so verbosely.  It's like using `$i` in a for loop.  If you've been programming more than a week you'll recognize the pattern in context.

If Rector can be told to chill on exception names, I'd favor being more lenient here.

Reply:
Not sure if this is possible, too less knowledge currently about all options. Will check....

Reply:
Ofc it is e.g. `$services->remove(\Rector\Doctrine\Rector\Class_\RemoveRepositoryFromEntityAnnotationRector::class)`. So I can still use predefined sets but remove single rules again.

Reply:
reverted